proto/prehog/v1alpha/tbot.proto

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package prehog.v1alpha;
+
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/gravitational/teleport/gen/proto/go/prehog/v1alpha";
+
+message TbotStartEvent {
```

Comment:
Can we please add a little comment on when this gets triggered?
Does it always send the event or only when the connection to teleport was successful?

Reply:
495c429865

Reply:
[495c429](https://github.com/gravitational/teleport/pull/22174/commits/495c429865a5f61963a06e9357a557d1c061ef16)